### DISCUSSION TITLE
text-wrap balance

### DISCUSSION THREAD

**Discussion by emmgfx** [tailwindlabs/tailwindcss#11427] [Ideas]
Hi all. I've just discovered the `text-wrap: balance;` property and not seen it in the docs. Would it be interesting to add this property?

Greetings!

**Comment by RobinMalfait** [comment]
Hey! We already have it in the insiders version and will be available in the next release. 

More info in the PR: https://github.com/tailwindlabs/tailwindcss/pull/11320

**Reply by emmgfx** [reply]
Thank you @RobinMalfait 👍

**Reply by mrleblanc101** [reply]
How do I install insiders ? 
And how do I use tailwindcss insiders with `@nuxtjs/tailwindcss` ?
I tried this in my package.json using `yarn`:
```json
  "resolutions": {
    "@nuxtjs/tailwindcss/tailwindcss": "insiders"
  }
  ```
  I keep getting:
  > Resolution field "insiders" has an invalid version entry and may be ignored

**Reply by wongjn** [reply]
You could look at installing Tailwindcss into Nuxt.js using [the instructions on the Tailwind website](https://tailwindcss.com/docs/guides/nuxtjs) which foregoes the `@nuxtjs/tailwindcss`, allowing you to install the insiders version.
